container-search/src/main/java/com/yahoo/prelude/semantics/SemanticSearcher.java

```diff
@@ -29,7 +30,7 @@
 
     private static final CompoundName rulesRulebase=new CompoundName("rules.rulebase");
     private static final CompoundName rulesOff=new CompoundName("rules.off");
-    private static final CompoundName tracelevelRules=new CompoundName("tracelevel.rules");
+    private static final CompoundName tracelevelRules=new CompoundName("trace.rules");
```

Comment:
This is an API change. I think it's ok though, but there's probably something you need to change in the doc.